packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -78,30 +84,93 @@ export class SCWSigner implements Signer {
     this.handleResponse(args, decrypted);
   }
 
-  async request(request: RequestArguments) {
+  async initSubAccountConfig() {
+    const config = store.subAccountsConfig.get();
+
+    if (!config?.enableAutoSubAccounts) {
+      throw new Error('called initSubAccountConfig but enableAutoSubAccounts is false');
+    }
+
+    // Get the owner account
+    const { account: owner } = config.toOwnerAccount
+      ? await config.toOwnerAccount()
+      : await getCryptoKeyAccount();
+
+    if (!owner) {
+      throw standardErrors.provider.unauthorized('No owner account found');
+    }
+
+    // Set the capabilities for the sub account
+    const capabilities = {
+      addSubAccount: {
+        account: {
+          type: 'create',
+          keys: [
+            {
+              type: owner.address ? 'address' : 'webauthn-p256',
+              key: owner.address || owner.publicKey,
+            },
+          ],
+        },
+      },
+    };
+
+    // Store the owner account and capabilities in the non-persisted config
+    store.subAccountsConfig.set({
+      capabilities,
+    });
+  }
+
+  // TODO: Properly type the return value
+  async request(request: RequestArguments): Promise<any> {
+    if (store.subAccountsConfig.get()?.enableAutoSubAccounts) {
+      await this.initSubAccountConfig();
+    }
```

Comment:
this will cause the popup blocker issue. We need to open the popup first. 

I think we'll want to refactor initializing into the request methods